app/views/shared/check_answers/_total_in_online_bank_accounts.html.erb

```diff
@@ -0,0 +1,21 @@
+
+<% online_current_accounts = @legal_aid_application.online_current_accounts_balance %>
+<% online_savings_accounts = @legal_aid_application.online_savings_accounts_balance %>
+
+<h3 class="govuk-heading-m"><%= t(".heading") %></h3>
```

Comment:
Looks like this has changed from `govuk-heading-m` and removed `print-no-break-after`. Is this intentional?